ami_assets/etc/xrd/xrd-vrouter-build-ami.sh

```diff
@@ -40,8 +40,10 @@ AMI_ASSETS="""
 /usr/lib/systemd/system/hugetlb-gigantic-pages.service
 """
 
-DPDK_SRC=${DPDK_SRC:-"https://fast.dpdk.org/rel/dpdk-19.11.12.tar.xz"}
-DPDK_DIR=${DPDK_DIR:-"dpdk-stable-19.11.12"}
+DPDK_KMODS_REPO=${DPDK_KMODS_REPO:-"git://dpdk.org/dpdk-kmods"}
+DPDK_KMODS_DIR=${DPDK_KMODS_DIR:-"dpdk-kmods"}
+DPDK_KMODS_VER=${DPDK_KMODS_VER:-"e721c733cd24206399bebb8f0751b0387c4c1595"}
```

Comment:
Is there any reason for picking this commit, other than its the first that contains the build instructions? (bit annoying that igb_uio is not versioned)

Reply:
I just picked the latest so we didn't get any unexpected updates.